flask/app/db.py

```diff
@@ -13,4 +13,4 @@
     )
 )
 
-client = MongoClient(MONGODB_URI)
+client = MongoClient(MONGODB_URI)
```

Comment:
I think Flake8 will complain about this

Reply:
forgot this one when we were doing the hardcoding, it's not replacing anything, just un-commenting.